spec/unit/plugins/digital_ocean_spec.rb

```diff
@@ -160,44 +154,13 @@
 
   describe "without digital_ocean hint file" do
     before do
-      allow(File).to receive(:exist?).with(hint_path_nix).and_return(false)
-      allow(File).to receive(:exist?).with(hint_path_win).and_return(false)
-    end
-
-    describe "with the /etc/digitalocean file" do
```

Comment:
The diff makes this look a bit odd, but we tested the same behavior with an EC2 file and with a DO file. We still have the same 2 checks we're just not checking with EC2 stuff anymore.